llvm/lib/Passes/PassRegistry.def

```diff
@@ -445,6 +445,7 @@ FUNCTION_PASS("transform-warning", WarnMissedTransformationsPass())
 FUNCTION_PASS("tsan", ThreadSanitizerPass())
 FUNCTION_PASS("memprof", MemProfilerPass())
 FUNCTION_PASS("declare-to-assign", llvm::AssignmentTrackingPass())
+FUNCTION_PASS("expand-large-div-rem", ExpandLargeDivRemPass(TM));
```

Comment:
This seems to already exist without the constructor argument in MachinePassRegistry.def, not sure what's going on there

Reply:
what do you mean?

```
$ ./build/rel/bin/opt -p expand-large-div-rem -S /dev/null
./build/rel/bin/opt: unknown pass name 'expand-large-div-rem'
```

Reply:
I mean this exists: https://github.com/llvm/llvm-project/blob/7b5505b0d597eddadd5ffd4bbf80a7e8804ec809/llvm/include/llvm/CodeGen/MachinePassRegistry.def#L46